quantum/plugins/ibm/accelerator/IBMAccelerator.cpp

```diff
@@ -221,6 +300,12 @@ void IBMAccelerator::initialize(const HeterogeneousMap &params) {
     backends_root = json::parse("{\"backends\":" + response + "}");
     getBackendPropsResponse = "{\"backends\":" + response + "}";
 
+    std::vector<std::string> your_available_backends;
+    selectBackend(your_available_backends);
+
```

Comment:
It's worth adding a `xacc::info` level logging here **if** the `lowest-queue-count` mode is enabled stating which actual backend was selected.
(since if we fail to select a backend in this mode, the below REST request will throw an error due to the invalid endpoint. It's easier to debug if we have a log here to catch that if needed)